linera-client/src/storage.rs

```diff
@@ -312,6 +312,8 @@ example service:tcp:127.0.0.1:7878:table_do_my_test"
             });
         }
         error!("available storage: memory");
+        #[cfg(feature = "storage-service")]
+        error!("Also available is linera-storage-service");
```

Comment:
Does it make sense for all of these features to be "enabled"? Or should some of them be mutually exclusive?

Reply:
Yes, it is possible to enable all of them.